test/common/markdown/markdown.tests.ps1

```diff
@@ -0,0 +1,114 @@
+$moduleRootFilePath = Split-Path -Path $PSScriptRoot -Parent
+
+# Identify the repository root path of the resource module
+$repoRootPath = (Resolve-Path -LiteralPath (Join-path $moduleRootFilePath "../..")).ProviderPath
+$repoRootPathFound = $false
+
+Describe 'Common Tests - Validate Markdown Files' -Tag 'CI' {
+    BeforeAll {
+        $skip = !$IsWindows
+        if ( !$skip ) 
+        {
+            $NpmInstalled = "not installed"
+            if (Get-Command -Name 'npm' -ErrorAction SilentlyContinue) 
+            {
+                $NpmInstalled = "Installed"
+                Write-Verbose -Message "NPM is checking Gulp is installed. This may take a few moments." -Verbose
```

Comment:
If Gulp is installed, will this step be a non-op? I'm interested in what would happen when running this on my dev Linux machine, I hope it won't reinstall anything in case I already have the dependencies installed.